src/audio/tdfb/tdfb_direction.c

```diff
@@ -556,7 +556,7 @@ void tdfb_direction_estimate(struct tdfb_comp_data *cd, int frames, int ch_count
 	 * to update user space.
 	 */
 	if (new_az_value != cd->az_value_estimate && time_since > CONTROL_UPDATE_MIN_TIME &&
-	    (cd->direction.trigger & 0x15) == 0x15) {
+	    (cd->direction.trigger & 0x0f) == 0x0f) {
```

Comment:
Probably best to have a macro mask here.

Reply:
Fixed now